src/Node.ts

```diff
@@ -5,12 +5,16 @@ import Config = require("./Config");
 
 export class Node {
     public data: number;
-    public next: Pointer;
+    public pointers: {
+        [key: string]: Pointer
+    };
```

Comment:
can we leave this out for now? 

I know it will be good to have in the future, but I think we are going to stick with just singly linked lists for at least the next 6 months or so. 

We are going to have to add a lot of complexity to the code base with allowing the way the diagram looks and the actions the user can take to be highly configurable, so it would be good to keep things simple where we can.